sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -757,7 +757,7 @@ public void validate() {
 
     private static final Duration KAFKA_POLL_TIMEOUT = Duration.millis(1000);
     // how long to wait for new records from kafka consumer inside start()
-    private static final Duration START_NEW_RECORDS_POLL_TIMEOUT = Duration.standardSeconds(5);
+    private static final Duration START_NEW_RECORDS_POLL_TIMEOUT = Duration.millis(10);
```

Comment:
Better to remove this then.

Recently when I modified KafkaIOTest, I removed a bit of extra code that handled 'false' from start(). I need to put that back. I can send a separate PR for that.